client/scss/utilities/_mixins.scss

```diff
@@ -127,7 +127,7 @@
   }
 }
 
-// Allow an element to be horizontally stretched outwith its container
+// Allow an element to be horizontally stretched out with its container
```

Comment:
'Outwith', one word, was what I'd meant, but I realise it's [a Scottish-ism](https://www.google.co.uk/search?q=outwith&oq=outwith&aqs=chrome..69i57j0l5.3866j0j7&sourceid=chrome&ie=UTF-8). Maybe 'outside of' would be better?